src/build-scripts/build_materialX_osl.py

```diff
@@ -14,15 +14,19 @@
 import os
 import sys
 import re
+import argparse
 
+# global defaults
+ARCH = 'linux64'
 MX_SOURCE = '../shaders/MaterialX'
-OSL_DEST = '../../dist/linux64/shaders/MaterialX'
+OSL_DEST = '../../dist/%s/shaders/MaterialX'%ARCH
```

Comment:
This should probably be 'build', not 'dest'.
The 'make install' should copy things from build to dest, but none of the build process should put things directly in dest.